documentation/book/cluster-operator.adoc

```diff
@@ -491,27 +491,32 @@ You can find more information on how to use it in the corresponding GitHub repo.
 For more information about using the metrics with Prometheus and Grafana, see <<_metrics>>
 
 ===== Logging
-The `logging` field allows the configuration of loggers. These loggers are:
-[source]
-log4j.rootLogger
-connect.root.logger.level
-log4j.logger.org.apache.zookeeper
-log4j.logger.org.I0Itec.zkclient
-log4j.logger.org.reflections
+The `logging` field allows the configuration of loggers. These loggers for zookeeper and kafka are available in `spec.zookeeper.logging` and `spec.kafka.logging` sections above respectively.
 
 The setting can be done in one of two ways. Either by specifying the loggers and their levels directly or by using a custom config map.
-An example of the latter would look like this::
+An examples would look like this::
+
+[source,yaml]
+----
+  logging:
+    type: inline
+    loggers:
+      logger.name: "VALUE"
+----
 
 [source,yaml]
 ----
   logging:
-    fromConfigMap:
-      name : customConfigMap
+    type: external
+    name: customConfigMap
 ----
 
+While using external config map remember to place your custom config map under `log4j.properties` key.
+
 The difference between these two options is that latter is not validated and does not support default values.
 That means user can supply any logging configuration, even if it is incorrect.
 First option supports default values.
+`VALUE` stands for one of these values: `INFO`, `ERROR`, `WARN`, `TRACE`, `DEBUG`, `FATAL` or `OFF`.
```

Comment:
This should be directly below the example. But I would argue that it's clearer to get rid of this template example and just use something a user could copy and paste, noting beneath that "The `INFO` could be replaced with any value log4j logger level: `INFO`, `ERROR`, `WARN`, `TRACE`, `DEBUG`, `FATAL` or `OFF`."